pkgs/by-name/lm/lmstudio/darwin.nix

```diff
@@ -10,8 +10,8 @@ stdenv.mkDerivation {
   inherit meta pname version;
 
   src = fetchurl {
-    url = "https://releases.lmstudio.ai/mac/arm64/${version}/b/latest/LM-Studio-${version}-arm64.dmg";
-    hash = "sha256-kb1XoTZjhCL1+CsV+r3/EN0srzIJ43H2VMZ779dVq1k=";
+    url = "https://releases.lmstudio.ai/mac/arm64/${version}/e/latest/LM-Studio-${version}-arm64.dmg";
```

Comment:
Would you mind, putting the letter into a variable "revision"? As [pointed out at lmstudio](https://github.com/lmstudio-ai/lmstudio-bug-tracker/issues/25), that scheme seems to be meant to last.